src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -33,6 +33,56 @@
 """ Space needed to reach the root of the literal block. """
 BASE_INDENT = " " * 12
 
+CATEGORIES_USED_ON_DESCRIPTION = [
```

Comment:
I believe this list should be sorted (probably easier to just call `.sort()` after the list definition).